apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.html.ts

```diff
@@ -152,52 +152,45 @@ export function InstructorAssessmentStatistics({
 
           <div class="card mb-4">
             <div class="card-header bg-primary text-white">
-              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Duration statistics
+              ${resLocals.assessment_set.name} ${assessment.number}: Duration statistics
             </div>
 
-            ${resLocals.assessment.score_stat_number > 0
+            ${assessment.score_stat_number > 0
               ? html`
                   <div class="card-body">
-                    <div id="durationHist" class="histogram"></div>
-                    <script>
-                      $(function () {
-                        var data = [${unsafeHtml(`${durationStat.hist}`)}];
-                        var xgrid = [${unsafeHtml(`${durationStat.threshold_seconds}`)}];
-                        var options = {
-                          ymin: 0,
-                          xlabel: 'duration',
-                          ylabel: 'number of students',
-                          xTickLabels: [
-                            ${unsafeHtml(
-                              `${durationStat.threshold_labels.map(function (label) {
-                                return JSON.stringify(label);
-                              })}`,
-                            )},
-                          ],
-                        };
-                        histogram('#durationHist', data, xgrid, options);
-                      });
-                    </script>
+                    <div
+                      id="durationHist"
+                      class="histogram"
+                      data-histogram="${JSON.stringify(durationStat.hist)}"
+                      data-xgrid="
+                      ${JSON.stringify(durationStat.threshold_seconds)}"
```

Comment:
Nit: this was broken into separate line?
```suggestion
                      data-xgrid="${JSON.stringify(durationStat.threshold_seconds)}"
```